UpstreamTagger/UpstreamTaggerHit.h

```diff
@@ -52,19 +52,20 @@ class UpstreamTaggerHit : public ShipHit
   private:
     UpstreamTaggerHit(const UpstreamTaggerHit& point);
     UpstreamTaggerHit operator=(const UpstreamTaggerHit& point);
-    Double_t v_drift = 17.7;//!    cm/ns
-    Double_t T_resol = 0.283; //!  Rpc time resolution in ns
+    Double_t v_drift = 17.7;// cm/ns
+    Double_t T_resol = 0.283; // Rpc time resolution in ns
 
-    Double_t point_final[3]; //[3]
-    const Double_t * mom[3]; //!
+    UpstreamTagger* c0;
```

Comment:
An UpstreamTagger object as a member of the Hit doesn't make sense. Also, it needs to be `//!` to make sure it's not serialised.

Reply:
The UpstreamTagger object is required to access its members inside GetNode() from UpstreamTaggerHit.cxx

Reply:
There are several other ways to do this.
UpstreamTaggerHit is the same format that will be used from data and for storage on disk. Having an UpstreamTagger member of this class (and serialising it), is unnecessary.

Again, I would suggest having a look at the implementation of similar methods for the strawtubes.